stock_picking_report_valued/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,12 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+===========================
+Stock Picking Report Valued
+===========================
+
```

Comment:
I think you should remove the first lines and just let the description. See https://github.com/OCA/maintainer-tools/blob/master/template/module/readme/DESCRIPTION.rst